domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -385,6 +389,18 @@ constructor(
                 .start(ContextCompat.getMainExecutor(application)) { videoRecordEvent ->
```

Comment:
nit: the main executor will probably be used anyways, but it's a good idea to use the calling coroutine's dispatcher as an executor here:
```
val callbackExecutor =
        (currentCoroutineContext()[ContinuationInterceptor] as? CoroutineDispatcher)?.asExecutor()
            ?: Runnable::run
```